src/models/subscription.ts

```diff
@@ -188,7 +189,8 @@ export class Subscription extends Model {
 				gitHubInstallationId: payload.installationId,
 				jiraHost: payload.host,
 				jiraClientKey: payload.hashedClientKey,
-				gitHubAppId: payload.gitHubAppId || null
+				gitHubAppId: payload.gitHubAppId || null,
+				avatarUrl: payload.avatarUrl || null
```

Comment:
Just to double check the install of `avatarUrl` is always `null` in the PR right? We can do that in a separate PR is that the plan?

Reply:
Yep leaving it as null, and we have a default avatar that we return when it is null. I dont think jira actually uses the avatar url atm, But at some point we should start storing it in another pr

Reply:
Update added the setting of the url in this pr.